### PR TITLE
Updates list of TPU-compatible ops for the TB graph viewer.

### PR DESCRIPTION
**Pull request by tayo**

### CODE REVIEW

**Comment by manivaradarajan** [comment]
Tayo, thanks for this. The team is in the midst of a complicated migration to Polymer 3 (#3887), which should finish soon. There will be some delay in getting to this PR.

**Comment by tayo** [comment]
@manivaradarajan Sure, no hurry.

File: tensorboard/plugins/graph/tf_graph_common/op.ts [hunk 1]
```diff
@@ -364,10 +471,17 @@ module tf.graph.op {
...
+      '_ArrayToList',
+      '_FusedBatchNormEx',
+      '_ListToArray',
       '_ParallelConcatUpdate',
+      '_RecvTPUEmbeddingActivations',
+      '_RecvTPUEmbeddingDeduplicationData',
       '_Retval',
       '_TPUCompile',
       '_TPUExecute',
+      '_SendTPUEmbeddingGradients',
```
Comment (psybuzz) [inline tensorboard/plugins/graph/tf_graph_common/op.ts:483]: nit: should `_Send...` should be alphabetcally sorted before L481?

File: tensorboard/plugins/graph/tf_graph_common/op.ts [hunk 2]
```diff
@@ -364,10 +471,17 @@ module tf.graph.op {
       'Switch',
       // Ops below are inserted by the compiler.
       '_Arg',
+      '_ArrayToList',
+      '_FusedBatchNormEx',
+      '_ListToArray',
       '_ParallelConcatUpdate',
```
Comment (psybuzz) [inline tensorboard/plugins/graph/tf_graph_common/op.ts:477]: When running `SupportedOpsMain` from `tensorflow/compiler/tf2xla/tf2xla_supported_ops.cc`, I'm not seeing these ops
- `_ParallelConcatUpdate`
- `_TPUCompile`
- `_TPUExecute`

in the list of results.  Is keeping them in the allowlist intentional?

File: tensorboard/plugins/graph/tf_graph_common/op.ts (hunk 1 above)
Comment (tayo) [reply tensorboard/plugins/graph/tf_graph_common/op.ts:483]: Done

File: tensorboard/plugins/graph/tf_graph_common/op.ts (hunk 2 above)
Comment (tayo) [reply tensorboard/plugins/graph/tf_graph_common/op.ts:477]: Yes.  The idea was to support 'legacy' existing graphs.  Although this policy can be changed later.

**Comment by stephanwlee** [comment]
The migration is done and we can now take in all the changes. Thanks for waiting.

Please rebase and update your pull request :)

**Review by stephanwlee — APPROVED** [review]
@psybuzz FYI, in order to incorporate the changes, I rebased against master, fixed conflict, then pushed to tayo's branch. 

Will merge when CI is happy.

**Comment by tayo** [comment]
Apologies for tardiness :). Thanks stephanwlee. Let me know if any other actions are necessary from my side.

**Pull request merged**